msg/versioned/VehicleOdometry.msg

```diff
@@ -1,34 +1,35 @@
-# Vehicle odometry data. Fits ROS REP 147 for aerial vehicles
+# Vehicle odometry data
+#
+# Fits ROS REP 147 for aerial vehicles
 
 uint32 MESSAGE_VERSION = 0
 
-uint64 timestamp		# time since system start (microseconds)
-uint64 timestamp_sample
+uint64 timestamp # [us] Time since system start
+uint64 timestamp_sample # [us] Timestamp sample
 
+uint8 pose_frame # [@enum POSE_FRAME] Position and orientation frame of reference
 uint8 POSE_FRAME_UNKNOWN = 0
-uint8 POSE_FRAME_NED     = 1 # NED earth-fixed frame
-uint8 POSE_FRAME_FRD     = 2 # FRD world-fixed frame, arbitrary heading reference
-uint8 pose_frame            # Position and orientation frame of reference
+uint8 POSE_FRAME_NED = 1 # NED earth-fixed frame. Origin is position of GC at startup.
+uint8 POSE_FRAME_FRD = 2 # FRD world-fixed frame. Arbitrary origin and heading reference. Z is down.
 
-float32[3] position         # Position in meters. Frame of reference defined by local_frame. NaN if invalid/unknown
-float32[4] q                # Quaternion rotation from FRD body frame to reference frame. First value NaN if invalid/unknown
+float32[3] position # [m] [@frame @local_frame] [@invalid NaN If invalid/unknown] Position
```

Comment:
We use `[@frame NED]` for frames, but I have invented a new syntax here, which is an @Xxxx to indicate something referenced in the document. 
When I generated docs this will cross link.

Are you OK with this because obviously someone reading the docs MIGHT not understand it? FYI @MaEtUgR